examples/step-48/step-48.cc

```diff
@@ -404,29 +407,32 @@ namespace Step48
     // solution. As in step-40, we need to equip the constraint matrix with
     // the IndexSet of locally relevant degrees of freedom to avoid it to
     // consume too much memory for big problems. Next, the <code> MatrixFree
-    // </code> for the problem is set up. Note that we specify the MPI
-    // communicator which we are going to use, and that we also want to use
-    // shared-memory parallelization (hence one would use multithreading for
-    // intra-node parallelism and not MPI; note that we here choose the
-    // standard option &mdash; if we wanted to disable shared memory
-    // parallelization, we would choose @p none). Finally, three solution
-    // vectors are initialized. MatrixFree stores the layout that is to be
-    // used by distributed vectors, so we just ask it to initialize the
-    // vectors.
+    // </code> object for the problem is set up. Note that we specify a
+    // particular scheme for shared-memory parallelization (hence one would
+    // use multithreading for intra-node parallelism and not MPI; we here
+    // choose the standard option &mdash; if we wanted to disable shared
+    // memory parallelization even in case there is more than one TBB thread
+    // available in the program, we would choose @p none). Also note that,
```

Comment:
Could you write the full type name `MatrixFree::AdditionalData::TasksParallelScheme::none` so that we get the link?